<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindow</class>
 <widget class="QMainWindow" name="AdminWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>500</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>481</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab_buscar">
     <attribute name="title">
      <string>Buscar</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>1001</width>
        <height>91</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>50</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>50</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_buscar">
         <property name="title">
          <string>Buscar</string>
         </property>
         <widget class="QWidget" name="horizontalLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>-1</x>
            <y>19</y>
            <width>451</width>
            <height>51</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <item>
            <widget class="QLineEdit" name="inputBuscar">
             <property name="placeholderText">
              <string>correo@gmail.com</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnBuscar">
             <property name="text">
              <string>Buscar</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_ordenar">
         <property name="title">
          <string>Ordenar</string>
         </property>
         <widget class="QWidget" name="horizontalLayoutWidget_3">
          <property name="geometry">
           <rect>
            <x>-1</x>
            <y>19</y>
            <width>451</width>
            <height>51</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="leftMargin">
            <number>10</number>
           </property>
           <property name="rightMargin">
            <number>10</number>
           </property>
           <item>
            <widget class="QComboBox" name="comboBoxOrdenar">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <item>
              <property name="text">
               <string>InOrden</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>PreOrden</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>PostOrden</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnOrdenar">
             <property name="text">
              <string>Aplicar</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>89</y>
        <width>1001</width>
        <height>351</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item row="0" column="0">
        <widget class="QTableWidget" name="tableUsuarios">
         <property name="editTriggers">
          <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>163</number>
         </attribute>
         <column>
          <property name="text">
           <string>Nombres</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Apellidos</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Correo</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Fecha de Nacimiento</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string/>
          </property>
         </column>
         <column>
          <property name="text">
           <string/>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_cargas">
     <attribute name="title">
      <string>Cargas Masivas</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget_4">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>-1</y>
        <width>1001</width>
        <height>451</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>30</number>
       </property>
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="topMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>20</number>
       </property>
       <property name="bottomMargin">
        <number>20</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Usuarios</string>
         </property>
         <widget class="QWidget" name="verticalLayoutWidget">
          <property name="geometry">
           <rect>
            <x>-1</x>
            <y>19</y>
            <width>301</width>
            <height>391</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="leftMargin">
            <number>20</number>
           </property>
           <property name="rightMargin">
            <number>20</number>
           </property>
           <item>
            <widget class="QPushButton" name="btnSubirUsuarios">
             <property name="layoutDirection">
              <enum>Qt::LayoutDirection::RightToLeft</enum>
             </property>
             <property name="text">
              <string>Cargar Usuarios </string>
             </property>
             <property name="icon">
              <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Solicitudes</string>
         </property>
         <widget class="QWidget" name="verticalLayoutWidget_2">
          <property name="geometry">
           <rect>
            <x>-1</x>
            <y>19</y>
            <width>301</width>
            <height>391</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="spacing">
            <number>30</number>
           </property>
           <property name="leftMargin">
            <number>20</number>
           </property>
           <property name="rightMargin">
            <number>20</number>
           </property>
           <item>
            <widget class="QPushButton" name="btnSubirSolicitudes">
             <property name="layoutDirection">
              <enum>Qt::LayoutDirection::RightToLeft</enum>
             </property>
             <property name="text">
              <string>Cargar Solicitudes</string>
             </property>
             <property name="icon">
              <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Publicaciones</string>
         </property>
         <widget class="QWidget" name="verticalLayoutWidget_3">
          <property name="geometry">
           <rect>
            <x>-1</x>
            <y>19</y>
            <width>301</width>
            <height>391</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="spacing">
            <number>30</number>
           </property>
           <property name="leftMargin">
            <number>20</number>
           </property>
           <property name="rightMargin">
            <number>20</number>
           </property>
           <item>
            <widget class="QPushButton" name="btnSubirPublicaciones">
             <property name="layoutDirection">
              <enum>Qt::LayoutDirection::RightToLeft</enum>
             </property>
             <property name="text">
              <string>Cargar Publicaciones</string>
             </property>
             <property name="icon">
              <iconset theme="QIcon::ThemeIcon::DocumentOpen"/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_reportes">
     <attribute name="title">
      <string>Reportes</string>
     </attribute>
     <widget class="QWidget" name="gridWidget_2" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1000</width>
        <height>450</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>30</number>
         </property>
         <property name="leftMargin">
          <number>20</number>
         </property>
         <property name="topMargin">
          <number>20</number>
         </property>
         <property name="rightMargin">
          <number>20</number>
         </property>
         <property name="bottomMargin">
          <number>20</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Arbol de Usuarios</string>
           </property>
           <widget class="QWidget" name="verticalWidget_4" native="true">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>20</y>
              <width>465</width>
              <height>340</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>10</number>
             </property>
             <property name="topMargin">
              <number>10</number>
             </property>
             <property name="rightMargin">
              <number>10</number>
             </property>
             <property name="bottomMargin">
              <number>10</number>
             </property>
             <item>
              <widget class="QLabel" name="img_usuarios">
               <property name="text">
                <string>IMAGEN</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_abrir_imgUsuarios">
               <property name="text">
                <string>Abrir Imagen</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string>Lista de Publicaciones</string>
           </property>
           <widget class="QWidget" name="verticalWidget_5" native="true">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>20</y>
              <width>465</width>
              <height>340</height>
             </rect>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>10</number>
             </property>
             <property name="topMargin">
              <number>10</number>
             </property>
             <property name="rightMargin">
              <number>10</number>
             </property>
             <property name="bottomMargin">
              <number>10</number>
             </property>
             <item>
              <widget class="QLabel" name="img_publicaciones">
               <property name="text">
                <string>IMAGEN</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignmentFlag::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_abrir_imgPosts">
               <property name="text">
                <string>Abrir Imagen</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QWidget" name="gridWidget_2" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>20</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QPushButton" name="btnGenerarReportes">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LayoutDirection::LeftToRight</enum>
            </property>
            <property name="text">
             <string>Generar Reporte</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LayoutDirection::RightToLeft</enum>
   </property>
   <widget class="QMenu" name="menuBienvenido">
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Bienvenido</string>
    </property>
    <addaction name="actionVista_Usuario"/>
    <addaction name="separator"/>
    <addaction name="actionCerrar_Sesion"/>
   </widget>
   <addaction name="menuBienvenido"/>
  </widget>
  <action name="actionCerrar_Sesion">
   <property name="text">
    <string>Cerrar Sesion</string>
   </property>
  </action>
  <action name="actionVista_Usuario">
   <property name="text">
    <string>Vista Usuario</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
